### PR TITLE
source-cal-com contribution from bishalbera

### LINKED ISSUE #47261: [connector-request] Cal.com
### Topic

This is a suggestion to create a connector for PandaDoc using the UI Connector Builder. The website for the connector is https://cal.com/.

### Relevant information

api docs: https://cal.com/docs/api-reference/v2/introduction

### PR DESCRIPTION
**Pull request by bishalbera**
## What

This PR adds a new connector for source Cal.com (source-cal-com).

Contributor-provided connector description:

The Cal.com connector enables seamless data synchronization between Cal.com’s scheduling platform and various destinations. It helps extract events, attendees, and booking details from Cal.com, making it easy to analyze scheduling data or integrate it into downstream systems like data warehouses or CRMs. This connector streamlines automated reporting and insights for time management and booking analytics
## Details

<details><summary>Expand for list of spec properties and streams</summary>

### Configuration

| Input | Type | Description | Default Value |
|-------|------|-------------|---------------|
| `orgId` | `string` | orgId.  |  |
| `api_key` | `string` | API Key. API key to use. Find it at https://cal.com/account |  |

### Streams
| Stream Name | Primary Key | Pagination | Supports Full Sync | Supports Incremental |
|-------------|-------------|------------|---------------------|----------------------|
| event_types | id | DefaultPaginator | ✅ |  ❌  |
| my_profile | id | No pagination | ✅ |  ❌  |
| schedules | id | DefaultPaginator | ✅ |  ❌  |
| calendars | externalId | No pagination | ✅ |  ❌  |
| bookings | id | DefaultPaginator | ✅ |  ❌  |
| conferencing | id | No pagination | ✅ |  ❌  |

</details>

## Reviewer checklist
- [ ] Run `/format-fix` to resolve any formatting errors
- [ ] Help the contributor upload an icon that follows our icon design guidelines
- [ ] Click into the CI workflows that wait for a maintainer to run them, which should trigger CI runs

### CODE REVIEW

File: airbyte-integrations/connectors/source-cal-com/icon.svg
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-cal-com/icon.svg:1]: This is very not square. 🙃

**Review by natikgadzhi — DISMISSED** [review]

**Comment by marcosmarxm** [comment]
Hello 👋 Due to the numerous contributions happening, I decided to create this [view](https://github.com/orgs/airbytehq/projects/104/views/3) to help you know when I plan to review your contribution. If your connector has some limited-time credential feel free to reach me out.

File: airbyte-integrations/connectors/source-cal-com/icon.svg
Comment (bishalbera) [reply airbyte-integrations/connectors/source-cal-com/icon.svg:1]: pardon me :( ...I've to edit the image

**Comment by marcosmarxm** [comment]
All contributions must be merged by Nov-14. I'm going to work to provide a first review by Friday to give users the weekend to fix any comment. Please be aware of this.

Make sure you have pagination properly working, the API doesn't have pagination please left a comment making my review easier.
Make sure have a square svg icon
Make sure credentials are valid.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
Missing offset pagination and limit.
Missing some important endpoints (teams, membership, users, conferencing)
You can remove slots and make the input more clear.

**Comment by bishalbera** [comment]
> Missing offset pagination and limit. Missing some important endpoints (teams, membership, users, conferencing) You can remove slots and make the input more clear.

these endpoints are under org feature which costs $37 @marcosmarxm thats why I wasnt able to implement them in the first place.

**Review by marcosmarxm — APPROVED** [review]

**Pull request merged**